Sunrise/Models/ProductProjection.swift

```diff
@@ -2,18 +2,10 @@
 // Copyright (c) 2016 Commercetools. All rights reserved.
 //
 
-import ObjectMapper
+import Commercetools
 
-struct ProductProjection: Mappable {
+extension ProductProjection {
 
-    // MARK: - Properties
-
-    var id: String?
-    var name: [String: String]?
-    var productTypeId: String?
-    var productType: ProductType?
-    var masterVariant: ProductVariant?
-    var variants: [ProductVariant]?
     /// The union of `masterVariant` and other`variants`.
     var allVariants: [ProductVariant] {
```

Comment:
Actually, I think `allVariants` could be a useful helper to others as well and we could move it to the SDK. Your choice 😉